scss/fontloader.scss

```diff
@@ -17,7 +17,7 @@
 
   // Replace the url to be local
   $web-font-path: str-replace($web-font-path,
-    'https://fonts.googleapis.com/css?family=', '../../vendor/fonts/typeface-');
+    'https://fonts.googleapis.com/css?family=', '../../../opensphere/vendor/fonts/typeface-');
```

Comment:
why does this work, but the current version not work in windows?

Reply:
This uses a direct path to `opensphere`, instead of a symlink in `<project>/vendor/fonts`. The issue was with symlinks not working on Windows.